src/containers/App/app.scss

```diff
@@ -1,7 +1,7 @@
 @import '../shared/css/variables';
 
 .app {
-  position: relative;
+  flex-grow: 1;
 
   .content {
     display: flex;
```

Comment:
You can get rid of the following because it is covered by `desktop-up`
```
   @include for-size(big-desktop-up) {
      min-height: $min-height;
    }
 ```